app/src/main/java/org/fossasia/openevent/general/OpenEventDatabase.kt

```diff
@@ -42,5 +50,9 @@ abstract class OpenEventDatabase : RoomDatabase() {
 
     abstract fun eventTopicsDao(): EventTopicsDao
 
+    abstract fun eventTypesDao(): EventTypesDao
+
+    abstract fun eventSubTopicDao(): EventSubTopicDao
```

Comment:
Since these are now stored with the event, this is unneeded

Reply:
but we can use dao in future for chips or filtering i suggest we should keep them

Reply:
We can't use them unless there is a table for them

Reply:
i'm saving them as well in a seperate table i could remove that part though but we may need them in future